build/load-balancer/Dockerfile

```diff
@@ -36,7 +36,7 @@ FROM ubuntu:20.10
 RUN apt-get update -y
 run apt-get install -y libmnl-dev
 run apt-get install -y libnetfilter-queue-dev
-RUN apt-get install -y iproute2 tcpdump iptables net-tools iputils-ping ipvsadm netcat
+RUN apt-get install -y iproute2 tcpdump iptables net-tools iputils-ping ipvsadm netcat ipset nftables
```

Comment:
is ipset needed? nftables sets are not ipsets

Reply:
No ipset is not needed, I added it when I was working on the version with iptables + ipset